plugins/europeana/entity.js

```diff
@@ -0,0 +1,53 @@
+import axios from 'axios';
+
+/**
+ * Get the entity data from the API
+ * @param {string} id of entity
+ * @param {string} type of entity
+ * @param {Object} params additional parameters sent to the API
+ * @param {string} params.wskey API key
+ * @return {Object} parsed record data
+ */
+function getEntity(id, type, params) {
+  return axios.get(`https://www.europeana.eu/api/entities/${getEntityTypeApi(type)}/base/${getEntityId(id)}`, {
+    params: {
+      wskey: params.wskey
+    }
+  })
+    .then((response) => {
+      return {
+        error: null,
+        data: response.data
```

Comment:
Let's be consistent with plugins/europeana/record.js and have the property here named `entity`.